armi/reactor/tests/test_reactors.py

```diff
@@ -44,26 +44,54 @@
 TEST_REACTOR = None  # pickled string of test reactor (for fast caching)
 
 
-def buildOperatorOfEmptyBlocks(customSettings=None):
+def buildOperatorOfEmptyBlocks(customSettings=None, reactorType="Hex"):
     """
     Builds a operator w/ a reactor object with some assemblies and blocks, but all are empty
 
     Doesn't depend on inputs and loads quickly.
+
+    Params
+    ------
+    customSettings : dict
+        Dictionary of off-default settings to update
+
+    reactorType : str
+        Str with value of ``Hex`` or `Cartesian``.
     """
+    if reactorType not in {"Hex", "Cartesian"}:
+        raise ValueError(f"{reactorType} is not supported by this method.")
     settings.setMasterCs(None)  # clear
     cs = settings.getMasterCs()  # fetch new
     cs["db"] = False  # stop use of database
     if customSettings is not None:
         cs.update(customSettings)
-    r = tests.getEmptyHexReactor()
+    if reactorType == "Hex":
+        r = tests.getEmptyHexReactor()
+        AssemClass = assemblies.HexAssembly
+        BlockClass = blocks.HexBlock
+        CompClass = Hexagon
+        dims = {"op": 16.0, "ip": 1}
+    elif reactorType == "Cartesian":
+        r = tests.getEmptyCartesianReactor()
+        AssemClass = assemblies.CartesianAssembly
+        BlockClass = blocks.CartesianBlock
+        CompClass = Rectangle
+        dims = {
+            "widthOuter": 16.0,
+            "lengthOuter": 10.0,
+            "widthInner": 1,
+            "lengthInner": 1,
+        }
```

Comment:
I think it might make sense to just make these totally independent without the input conditional/switch, even though they share some code.

Reply:
yeah, i had that thought after I got near the end.